src/vs/workbench/contrib/debug/browser/breakpointsView.ts

```diff
@@ -673,6 +690,71 @@ class DataBreakpointsRenderer implements IListRenderer<DataBreakpoint, IDataBrea
 	}
 }
 
+class InstructionBreakpointsRenderer implements IListRenderer<IInstructionBreakpoint, IInstructionBreakpointTemplateData> {
+
+	constructor(
+		@IDebugService private readonly debugService: IDebugService,
+		@ILabelService private readonly labelService: ILabelService
+	) {
+		// noop
+	}
+
+	static readonly ID = 'instructionBreakpoints';
+
+	get templateId() {
+		return InstructionBreakpointsRenderer.ID;
+	}
+
+	renderTemplate(container: HTMLElement): IInstructionBreakpointTemplateData {
```

Comment:
we need a new icon for instruction breakpoints.

Reply:
@iidorn -> @isidorn  ;-)

Reply:
@jogo- thanks for pointing that out